.github/actions/e2e/action.yml

```diff
@@ -0,0 +1,95 @@
+name: e2e-parallel
+description: "Trigger e2e tests and create reports"
+inputs:
+  apiUrl:
+    description: "API URL from PR or staging"
+    required: true
+  baseUrl:
+    description: "FE URL from PR or staging"
+    required: true
+  commitInfoMessage:
+    description: "Info displayed in commit"
+    required: false
+  cypressGrepTags:
+    description: "Passed tags that should be executed"
+    required: true
+  split:
+    description: "E2E tests containers number"
+    required: true
+  splitIndex:
+    description: "E2E tests container index"
+    required: true
+  install:
+    description: "Triggers cypress installation"
+    required: false
+  browser:
+    description: "Browser which e2e should test"
+    required: false
+  appMountUri:
+    description: "URL for Apps"
+    required: true
+  userName:
+    description: "Name of user which e2e uses"
+    required: true
+  secondUserName:
+    description: "Name of the second user which e2e uses"
+    required: true
+  userPassword:
+    description: "User password for e2e"
+    required: true
+  permissionsUserPassword:
+    description: "User with limited permissions password"
+    required: true
+  githubToken:
+    description: "GH token"
+    required: false
```

Comment:
Where we are using value from this input?

Reply:
.github/workflows/tests-nightly.yml 180 line

Reply:
I did not validate inputs in workflows, I just refactored what we had - might be it is not needed

Reply:
@wojteknowacki I don't see the token being used in this file (`.github/actions/e2e/action.yml`), probably a leftover?

Also it seems like we don't need to pass it, you can retrieve it from `${{ github.token }}` inside actions (https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow)